activetable/activetable.py

```diff
@@ -570,6 +577,58 @@ def generate_style_sheet(self, stylesheet):
         sample_style_sheet.alignment = 1
         return sample_style_sheet
 
+    def get_student_item_dict(self, user=None):
+        """
+        Returns dict required by the submissions app for creating and
+        retrieving submissions for a particular student.
+        """
+        if user is None:
+            user = self.runtime.get_real_user(self.runtime.anonymous_student_id)
+
+        return {
+            "student_id": user.id,
+            "course_id": unicode(self.course_id),
+            "item_id": unicode(self.scope_ids.usage_id),
+            "item_type": ITEM_TYPE,
+        }
+
+    @staticmethod
+    def custom_report_format(*args, **kwargs):
+        """
+        This returns a html string with the activatable answers for the given user and block.
+        """
+        user = kwargs.get("user")
+        block = kwargs.get("block")
+
+        answers = {}
+
+        if not block:
+            return ''
+        elif user:
+            student_item_dict = block.get_student_item_dict(user=user)
+            submission = api.get_submissions(student_item_dict, limit=1)
+            try:
+                user_answers = submission[0]["answer"]
+                answers = user_answers.get("answers", {})
+            except IndexError:
+                pass
+
+        block.answers = answers
+        block.parse_fields()
+        block.postprocess_table()
+
+        context = dict(
+            total_width=sum(block._column_widths) if block._column_widths else None,
+            column_widths=block._column_widths,
+            head_height=block._row_heights[0] if block._row_heights else None,
+            thead=block.thead,
+            tbody=block.tbody,
+            score_type=block.score_type,
+            headers_style=re.sub("<[^<]*table>", "", block.headers_style) if block.custom_headers else None,
+        )
+        html = loader.render_template('templates/html/activetable_custom_report_format.html', context)
+        return html.replace('\n', "")
```

Comment:
Please don't mix single quotes and double quotes, use only one of them. @andrey-canon